docs/powered_by.rst

```diff
@@ -11,3 +11,4 @@ make a Pull Request!
 * https://www.farmersbusinessnetwork.com
 * `Home Assistant <https://home-assistant.io>`_
 * KeepSafe https://www.getkeepsafe.com/
+* Skyscanner Hotels https://www.skyscanner.net/hotels
```

Comment:
Please replace with `SkyScanner` to make our docs spellchecker happy.
Or add `Skyscanner` word to https://github.com/aio-libs/aiohttp/blob/master/docs/spelling_wordlist.txt